draft-ietf-quic-transport.md

```diff
@@ -2400,15 +2423,11 @@ used to amplify the volume of data that an attacker can generate toward a
 victim.
 
 As described in {{migration-response}}, an endpoint is required to validate a
-peer's new address to confirm the peer's possession of the new address.  Until a
-peer's address is deemed valid, an endpoint MUST limit the rate at which it
-sends data to this address.  The endpoint MUST NOT send more than a minimum
-congestion window's worth of data per estimated round-trip time (kMinimumWindow,
-as defined in {{QUIC-RECOVERY}}).  In the absence of this limit, an endpoint
-risks being used for a denial of service attack against an unsuspecting victim.
-Note that since the endpoint will not have any round-trip time measurements to
-this address, the estimate SHOULD be the default initial value; see
-{{QUIC-RECOVERY}}.
+peer's new address to confirm the peer's possession of the new address. Until a
+peer's address is deemed valid, an endpoint limits the amount of data it sends
+to that address; see {{address-validation}}. In the absence of this limit, an
+endpoint risks being used for a denial of service attack against an
+unsuspecting victim.
```

Comment:
This does introduce some interesting questions around NAT rebinding if the "new address" was only sending very small ACK-only packets, but that should only persist for the single RTT it takes to get the PATH_RESPONSE back. Probably fine to leave the wording here as-is and not call that out, but thought I'd double check.